hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -97,13 +97,13 @@
                    the FIFOs, the CDC's, the core state machine and the shift register.
                    In the current implementation, the CDC FIFOs are drained not reset.
                    Therefore software must confirm that both FIFO's empty before releasing
-                   clearing this bit.''',
+                   this bit.''',
```

Comment:
IMHO, "releasing this bit" sounds a little unusual.  I'd probably say one of:

a) "... before clearing this bit."
b) "... before releasing the IP from reset."

Reply:
Done!